app/src/main/java/com/example/concordia_campus_guide/Activities/SearchActivity.java

```diff
@@ -47,23 +42,19 @@ protected void onCreate(Bundle savedInstanceState) {
         //setting up activity
         setContentView(R.layout.search_activity);
         setBackButtonOnClick();
-        searchResults = (ListView) findViewById(R.id.searchResults);
-        searchText = (EditText) findViewById(R.id.searchText);
-        nextClassText = (TextView) findViewById(R.id.next_class_text);
-        nextClassArrow = (ImageView) findViewById(R.id.next_class_arrow);
-        nextClassRow = (View) findViewById(R.id.view_container);
-        calendarViewModel = ViewModelProviders.of(this).get(CalendarViewModel.class);
-
+        searchResults = findViewById(R.id.searchResults);
+        searchText = findViewById(R.id.searchText);
+        nextClassText = findViewById(R.id.next_class_text);
+        nextClassArrow = findViewById(R.id.next_class_arrow);
+        nextClassRow = findViewById(R.id.view_container);
+        calendarViewModel = new ViewModelProvider(this).get(CalendarViewModel.class);
```

Comment:
hey please could you put all the inits in an initComponent() please? it will be cleaner!

Reply:
great idea, will do!